package/gluon-respondd/files/etc/init.d/gluon-respondd

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh /etc/rc.common
+
+EXTRA_COMMANDS='restart_if_running'
+
+START=50
+
+SERVICE_WRITE_PID=1
+SERVICE_DAEMONIZE=1
+
+DEVLIST=/var/run/gluon-respondd.devs
+DAEMON=/usr/bin/respondd
+LOCK=/var/run/gluon-respondd.lock
+
+
+do_start() {
+	DEVS=$(cat $DEVLIST 2>/dev/null | while read dev iface; do echo -n " -i $dev"; done)
```

Comment:
No need to use `cat` here, simply do `while read dev iface; do ...; done < $DEVLIST`